sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -240,50 +229,52 @@ public void run() {
 				}
 			});
 		}
-
-		String deviceID = AuthRepository.getInstance().getDeviceID();
```

Comment:
we need to keep the device_id for backward compatibility and continues BI. How about use device_id and  app_device_id same as user_id and app_user_id?

Reply:
not sure, because its an overhead to keep both. I'll talk with Oded about it.